### ISSUE TITLE
Does not work with TightVNC binaries

### ISSUE THREAD

**Issue by CaledoniaProject** [TheWover/donut#39]
Here we are again :-)

To reproduce this issue, [install TightVNC 2.8.27 from here](https://www.tightvnc.com/download.php). Once finished, configure the port and start listening. This is to confirm the configuration is working.

Then kill the process and try to load it with donut:

* [donut.txt](https://github.com/TheWover/donut/files/4209230/donut.txt)
* [loader.txt](https://github.com/TheWover/donut/files/4209231/loader.txt)

And nothing happens

**Comment by odzhan** [comment]
Hello again :wink:

I installed TVN and confirmed the donut loader crashes when trying to run tvnserver.exe. I don't know the exact reason right now, but I observed svchost in process monitor trying to open loader.exe which may have something to do with why a crash occurs. Then I tried to run the loader under the SYSTEM account and it does appear to listen for incoming connections only to crash later once a client connects. I'm open to correction, but as I see it, this is a problem where tvnserver.exe is expected to be a file on disk and not in memory, similar to cygwin binaries and mixed assemblies.

Thanks for reporting the issue. However, I don't know when or if this will ever be fixed. 
I'd recommend a VNC server implemented as a DLL, [something like this](https://github.com/rsmudge/vncdll).
